### PR TITLE
Update VideoChip.md

### PR DESCRIPTION
**Pull request by DotEXEGaming**
Adding the new methods RasterCustomSprite and DrawCustomSprite introduced by RG 0.1.3.

### CODE REVIEW

File: en/docs/modules/misc/VideoChip.md [hunk 1]
```diff
@@ -88,6 +88,10 @@ end
 
 <img src="../../../assets/docs/VideoChip/SpriteFont.png" width="200" align="right">
 
+⚠️ **This feature is new, as of Retro Gadgets 0.1.3, and not fully tested yet**
+### DrawCustomSprite(position `vec2`, spriteSheet `SpriteSheet`, spriteOffset `vec2`, spriteSize `vec2`, tintColor `color`, backgroundColor `color`)
```
Comment (kurmachu) [inline en/docs/modules/misc/VideoChip.md:92]: These lines should be rearranged to follow the Method Name -> Method Details convention.
```suggestion
### DrawCustomSprite(position `vec2`, spriteSheet `SpriteSheet`, spriteOffset `vec2`, spriteSize `vec2`, tintColor `color`, backgroundColor `color`)
⚠️ **This feature is new, as of Retro Gadgets 0.1.3, and not fully tested yet**
```

File: en/docs/modules/misc/VideoChip.md [hunk 2]
```diff
@@ -108,6 +112,10 @@ end
 ### RasterSprite(position1 `vec2`, position2 `vec2`, position3 `vec2`, position4 `vec2`, spriteSheet `SpriteSheet`, spriteX `number`, spriteY `number`, tintColor `color`, backgroundColor `color` )
 Draws **an entire sprite sheet** mapping it on a quad identified by position1, position2, position3, and position4. The locations of `position1`, `position2`, `position3` and `position4` are the top-left, top-right, bottom-right and bottom-left respectively. 
 
+⚠️ **This feature is new, as of Retro Gadgets 0.1.3, and not fully tested yet**
+### RasterCustomSprite(position1 `vec2`, position2 `vec2`, position3 `vec2`, position4 `vec2`, spriteSheet `SpriteSheet`, spriteOffset `vec2`, spriteSize `vec2`, tintColor `color`, backgroundColor `color`)
```
Comment (kurmachu) [inline en/docs/modules/misc/VideoChip.md:116]: These lines should be rearranged to follow the Method Name -> Method Details convention.
```suggestion
### RasterCustomSprite(position1 `vec2`, position2 `vec2`, position3 `vec2`, position4 `vec2`, spriteSheet `SpriteSheet`, spriteOffset `vec2`, spriteSize `vec2`, tintColor `color`, backgroundColor `color`)
⚠️ **This feature is new, as of Retro Gadgets 0.1.3, and not fully tested yet**
```

**Review by kurmachu — CHANGES_REQUESTED** [review]

**Comment by DotEXEGaming** [comment]
Commited both change requests.

**Review by kurmachu — CHANGES_REQUESTED** [review]
Image for draw text is misplaced.
image

**Comment by DotEXEGaming** [comment]
Fixed.

**Comment by DotEXEGaming** [comment]
Fixed the weird S hanging around.

File: en/docs/modules/misc/VideoChip.md [hunk 3]
```diff
@@ -86,6 +86,10 @@ end
 ```
 <img src="../../../assets/docs/VideoChip/ExampleImageAsset.png" style="max-width: 400px">
 
+### DrawCustomSprite(position `vec2`, spriteSheet `SpriteSheet`, spriteOffset `vec2`, spriteSize `vec2`, tintColor `color`, backgroundColor `color`)
+⚠️ **This feature is new, as of Retro Gadgets 0.1.3, and not fully tested yet**
```
Comment (kurmachu) [inline en/docs/modules/misc/VideoChip.md:90]: Add a period to end sentence, and two spaces to allow newline in markdown.
```suggestion
⚠️ **This feature is new, as of Retro Gadgets 0.1.3, and not fully tested yet.**  
```

File: en/docs/modules/misc/VideoChip.md [hunk 4]
```diff
@@ -108,6 +112,10 @@ end
 ### RasterSprite(position1 `vec2`, position2 `vec2`, position3 `vec2`, position4 `vec2`, spriteSheet `SpriteSheet`, spriteX `number`, spriteY `number`, tintColor `color`, backgroundColor `color` )
 Draws **an entire sprite sheet** mapping it on a quad identified by position1, position2, position3, and position4. The locations of `position1`, `position2`, `position3` and `position4` are the top-left, top-right, bottom-right and bottom-left respectively. 
 
+### RasterCustomSprite(position1 `vec2`, position2 `vec2`, position3 `vec2`, position4 `vec2`, spriteSheet `SpriteSheet`, spriteOffset `vec2`, spriteSize `vec2`, tintColor `color`, backgroundColor `color`)
+⚠️ **This feature is new, as of Retro Gadgets 0.1.3, and not fully tested yet**
```
Comment (kurmachu) [inline en/docs/modules/misc/VideoChip.md:116]: (2nd) Add a period to end sentence, and two spaces to allow newline in markdown.
```suggestion
⚠️ **This feature is new, as of Retro Gadgets 0.1.3, and not fully tested yet.**  
```

**Review by kurmachu — CHANGES_REQUESTED** [review]
So close! Just a tiny touch up, and you're good to go!

**Comment by DotEXEGaming** [comment]
Commited the corrections.

**Review by kurmachu — APPROVED** [review]

**Pull request merged**